openebs/pkg/v1/maya_api.go

```diff
@@ -117,12 +112,12 @@ func (v OpenEBSVolume) CreateVsm(vname, size, storageClassName string) (string,
 		return "HTTP Status error from maya-apiserver", err
 	}
 
-	glog.Infof("VSM Successfully Created:\n%v\n", string(data))
-	return "VSM Successfully Created", nil
+	glog.Infof("Volume Successfully Created:\n%v\n", string(data))
```

Comment:
If you are pushing the entire data then it should be a DEBUG. If just the name of volume, then INFO